syntax/output.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include "tokens.hpp"
+
+using namespace std;
+
+static const char *token_to_string(int tok) {
+    switch (tok) {
+    case 0: return "EOF"; break;
+    case Fnc: return "FNC"; break;
+    case Extern: return "EXTERN"; break;
+    case Operator: return "op"; break;
+    case Operator_kw: return "OPERATOR"; break;
+    case Include: return "INCLUDE"; break;
+    case Type: return "TYPE"; break;
+    case Ref: return "REF"; break;
+    case Val: return "VAL"; break;
+    case Implement: return "IMPLEMENT"; break;
+    case For: return "FOR"; break;
+    case Destructor: return "DESTRUCTOR"; break;
+    case If: return "IF"; break;
+    case While: return "WHILE"; break;
+    case Else: return "ELSE"; break;
+    case Ret: return "RET"; break;
+    case Generic: return "GENERIC"; break;
+    case Bool: return "bool"; break;
+    case Int: return "int"; break;
+    case Str: return "str"; break;
+    case Float: return "float"; break;
+    case Ident: return "ID"; break;
+
+    case Eq: return "="; break;
+    case '(': return "'('"; break;
+    case ')': return "')'"; break;
+    case ';': return "';'"; break;
+    case ',': return "','"; break;
+    case '{': return "'{'"; break;
+    case '}': return "'}'"; break;
+
+    default: return "<INVALID TOKEN>";
+    }
```

Comment:
I'll do this on the next step.

Also this is for tokens of type `Error`, and bison would just report them as erroneous.
Since we don't have bison yet, we mark them as invalid by ourselves.